From: Bernhard Walle <bernhard@bwalle.de>
Date: Sun, 19 Feb 2012 15:10:28 +0100
Subject: [PATCH] dockstar: Adjust default environment

Mount the root file system readwritable and don't load an initrd.

Signed-off-by: Bernhard Walle <bernhard@bwalle.de>
---
 include/configs/dockstar.h |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/include/configs/dockstar.h b/include/configs/dockstar.h
index 249f93b..cf82ea0 100644
--- a/include/configs/dockstar.h
+++ b/include/configs/dockstar.h
@@ -87,8 +87,7 @@
 	"ubi part root; " \
 	"ubifsmount root; " \
 	"ubifsload 0x800000 ${kernel}; " \
-	"ubifsload 0x1100000 ${initrd}; " \
-	"bootm 0x800000 0x1100000"
+	"bootm 0x800000"
 
 #define CONFIG_MTDPARTS		"mtdparts=orion_nand:1m(uboot),-(root)\0"
 
@@ -97,8 +96,7 @@
 	"mtdids=nand0=orion_nand\0" \
 	"mtdparts="CONFIG_MTDPARTS \
 	"kernel=/boot/uImage\0" \
-	"initrd=/boot/uInitrd\0" \
-	"bootargs_root=ubi.mtd=1 root=ubi0:root rootfstype=ubifs ro\0"
+	"bootargs_root=ubi.mtd=1 root=ubi0:root rootfstype=ubifs rw\0"
 
 /*
  * Ethernet Driver configuration
